src/components/Marketplace/AddListing.js

```diff
@@ -0,0 +1,313 @@
+    {
+      value: 'pallet',
+      label: 'Pallet',
+    },
+  ];
+
+  const Seasons = [
+    {
+      value: 'Summer',
+      label: 'Summer',
+    },
+    {
+      value: 'Winter',
+      label: 'Winter',
+    },
+    {
+      value: 'Fall',
+      label: 'Fall',
+    },
+    {
+      value: 'Spring',
+      label: 'Spring',
+    },
+  ];
+  return (
+    <form id="listing-form" onSubmit={createOrEditRecord}>
+      <Grid spacing={3} container className="listing-form">
+        <Grid item xs={12}>
+          <Typography variant="h5" component="h5" className={classes.infoDivider}>Basic Information</Typography>
+        </Grid>
+        <Grid item xs={12}>
+          <ListingInputField
+            label="Product Name"
+            name="crop"
+            onChange={onChangeField}
+            val={ListingRecord.crop}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <ListingInputField
+            label="Product Description"
+            name="description"
+            onChange={onChangeField}
+            val={ListingRecord.description}
+            multiline
+          />
+        </Grid>
+        <Grid item container xs={12}>
+          <Typography variant="h5" component="h5" className={classes.infoDivider}>Units</Typography>
+        </Grid>
+        <Grid item xs={4}>
+          <FormControl className={classes.inputField} variant="outlined">
+            <Typography variant="h6" component="h4">
+              Unit Type
+            </Typography>
+            <Select
+              name="unit type"
+              onChange={onChangeField}
+              defaultValue=""
+              value={ListingRecord['unit type']}
+              required
+            >
+              {UnitTypes.map((name) => (
+                <MenuItem value={name.value}>
+                  {name.label}
+                </MenuItem>
+              ))}
+
+            </Select>
+          </FormControl>
+        </Grid>
+        <Grid item xs={4}>
+          <ListingInputField
+            id="standard-number"
+            label="Units per Pallet"
+            name="units per pallet"
+            type="number"
+            onChange={onChangeField}
+            val={ListingRecord['units per pallet']}
+          />
+        </Grid>
+        <Grid item xs={4}>
+          <ListingInputField
+            id="standard-number"
+            label="Pounds per Unit"
+            name="lbs per unit"
+            type="number"
+            onChange={onChangeField}
+            val={ListingRecord['lbs per unit']}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <Typography variant="h5" component="h5" className={classes.infoDivider}>Prices</Typography>
+          <ListingInputField
+            id="standard-number"
+            label="Standard Price per Unit"
+            name="standard price per unit"
+            type="number"
+            onChange={onChangeField}
+            val={ListingRecord['standard price per unit']}
+          />
```

Comment:
Can we also add another text input right next to this that corresponds with the Airtable attribute, restricted price per pallet? We would have to change the Airtable attribute name to that as well, but I don't want to do that at this moment in case it breaks your feature. We just had a meeting with F2P where they specified that farmers should be able to give discounts to agencies as well!